### ISSUE TITLE
Using with Sage and Doctrine/Inflector dependency conflict

### ISSUE THREAD

**Issue by creativecodeology** [StoutLogic/acf-builder#69]
Hello, I'm using the Roots/Bedrock/Sage framework, and while installing acf-builder using composer, I ran into the issue of a dependency conflict with Doctrine/Inflector. Sage requires Doctrine/Inflector at least 1.3, and acf-builder requires ~1.1.0.

```
roots/sage              dev-master  requires  doctrine/inflector (^1.3)
stoutlogic/acf-builder  1.6.1       requires  doctrine/inflector (~1.1.0)
```

To test, I aliased doctrine/inflector in composer.json to "pretend" to be 1.1.0, and this worked until inflector was called for a repeater field, and this is where it breaks.

Is there any way to resolve this conflict between Sage/acf-builder?

**Comment by stevep** [comment]
So my composer-fu versioning is failing me at the moment. I need to support < 1.3 because 1.3 requires php 7.1 but I still want to support php 5.6 with this library. How can I alter the composer.json file to allow for this?

Also I'm having a problem with packagist.com updating versions. I did release 1.7.0 which removed the composer.lock file and then a 1.7.1 which updated the composer.json file with the version and someone said that fixed a similar problem. But packagist isn't picking it up. Can you try dev-master of ACF Builder and see that fixes your issue first?

https://github.com/StoutLogic/acf-builder/compare/1.7.0...master

**Comment by creativecodeology** [comment]
I tried, packagist doesn't even show the ~1.7.0 versions. I then attempted to load from the git repository directly with composer, and again, composer said it couldn't locate 1.7.1. Fed up with composer, I cloned the repository to /vendor/acf-builder, and required the dependencies manually using autoload.php. This did work as in recognizing the class and my fields shown, but again the same issue as soon as I add a repeater field. I tried debugging using xdebug, but vscode is giving me some problems with xdebug, so once I figure that out I'll share the results here.

**Comment by stevep** [comment]
If you put `dev-master` as the version for acf builder in your composer.json that should work for now, though not good long term.

**Comment by stevep** [comment]
Made [this](https://github.com/StoutLogic/acf-builder/commit/43d1b68100b5e0d30b238d5bf710bf06299694f9) change to the composer.json and the Travis CI tests passed, try dev master and see if that resolves your issues and then I'll work on packagist.

**Comment by creativecodeology** [comment]
Thank you, I do think that helped. Ultimately however, I ran into another issue once the dependency conflict was resolved. It appears if you put a repeater field into a tabbed section, unless you add ```->endRepeater()``` it will fail, and it was notoriously difficult to debug this as it wasn't throwing any errors or exceptions, just crashing.

**Comment by QWp6t** [comment]
```json
  "require": {
    "doctrine/inflector": "1.3 as 1.1"
  }
```

**Comment by stevep** [comment]
Packagist is fixed and 1.7.2 has been released with a more relaxed doctrine/inflector version requirement.
